whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
 %-
 % ====================================================================
 
+\newcommand{\red}[1]{\textcolor{red}{#1}}
+
 \section{Weak Lensing}
 \def\secname{wl}\label{sec:\secname}
 
-\credit{StephenRidgway},
 \credit{jmeyers314},
-\credit{tonytyson}.
+\credit{tonytyson},
+\credit{StephenRidgway}.
 
-Much of LSST cosmology may be limited by systematic errors rather than
-photon signal-to-noise. This is especially true of weak gravitational
-lensing,  which relies on very accurate (\ie low bias), but very low
-signal-to-noise, measurements of the shapes of galaxies, and high
-signal-to-noise measurements of PSF calibration stars. As outlined in the SRD,
-uniformity of seeing in the bands used for WL and special observing strategies 
-are required in order to reduce additive and multiplicative shear systematics.
+Much of LSST cosmology may be limited by systematic rather than statistical
+errors.  This is especially true of weak gravitational lensing, which relies on
+very accurate (\ie low bias), but very noisy, measurements of the shapes of many
+galaxies, and high signal-to-noise measurements of PSF calibration stars.  As
+outlined in the SRD, uniformity of seeing in the bands used for weak lensing and
+special observing strategies are required in order to reduce additive and
+multiplicative shear systematics.
 
 Achieving the ultimate sensitivity of the LSST to weak lensing science places
-stringent requirements on our ability to accurately measure galaxy shapes and redshifts,
-which in turn demands precise and accurate knowledge of the point spread function,
-astrometry, and photometry. These measurements are influenced by the interaction of
-light with the Earth’s atmosphere, the telescope optics, and the CCD sensors. Sysematics
-in the shear are introduced in each case.   Methods have been developed for suppressing 
-these systematics in current lensing surveys. These and new methods will be applied to 
-the LSST survey.
-
-Over the sample of 3-4 billion galaxies, the shear systematics must be below
-one part in 10,000 for additive shear, and one part in 1000 for multiplicative shear.
-Each visit to a sky patch encounters these systematics. Some observing strategies can 
-effectively randomize these over all visits to a field.  Below we discuss the observing 
-strategies for suppressing shear systematics and metrics for their success.
+stringent requirements on our ability to accurately measure galaxy shapes and
+redshifts, which in turn demands precise and accurate knowledge of the point
```

Comment:
Yes.  Our uniformity discussion should be expanded, since this drives 
the uniformity metric in the scheduler.

On 6/29/16 6:44 PM, Rachel Mandelbaum wrote:
> In whitepaper/Cosmology/wl.tex
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478#discussion_r69057927>:
>
>>
>>  Achieving the ultimate sensitivity of the LSST to weak lensing science places
>> -stringent requirements on our ability to accurately measure galaxy shapes and redshifts,
>> -which in turn demands precise and accurate knowledge of the point spread function,
>> -astrometry, and photometry. These measurements are influenced by the interaction of
>> -light with the Earth’s atmosphere, the telescope optics, and the CCD sensors. Sysematics
>> -in the shear are introduced in each case.   Methods have been developed for suppressing
>> -these systematics in current lensing surveys. These and new methods will be applied to
>> -the LSST survey.
>> -
>> -Over the sample of 3-4 billion galaxies, the shear systematics must be below
>> -one part in 10,000 for additive shear, and one part in 1000 for multiplicative shear.
>> -Each visit to a sky patch encounters these systematics. Some observing strategies can
>> -effectively randomize these over all visits to a field.  Below we discuss the observing
>> -strategies for suppressing shear systematics and metrics for their success.
>> +stringent requirements on our ability to accurately measure galaxy shapes and
>> +redshifts, which in turn demands precise and accurate knowledge of the point
>
> Another nitpicky point - we can tolerate photo-z error as long as it is
> well known/characterized. The issue with survey inhomogeneity is the
> difficulty in characterizing photo-z error trends (and a strong
> variation in those trends with position). If our survey strategy were to
> simply make the photo-z error slightly larger in a completely uniform
> and well-understood way, that would not be a problem.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478/files/68564bd57311f22658256026e5e6e6a39c2554f4#r69057927>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACA6Yp_AI9VLPXmOI8vT2QekC-E2rX95ks5qQx-SgaJpZM4I9hd->.
>